playwright-home-task-1/test-2-add-product-to-the-cart.spec.js

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('Verify Shopping Cart icon contains the number of products added', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
```

Comment:
We can move this url to https://github.com/KristinaMytsyk/playwright_swag_labs/pull/2/files#diff-8421b2f4ad242eb670ea84ae0057d8708f0370adad5ac063da20b8abc54426daR28